css-text/Overview.bs

```diff
@@ -1324,6 +1327,11 @@ Canonical order: N/A
       Shaping characters are still shaped as if the word were not
       broken, and grapheme clusters must stay together as one unit.
       No hyphenation character is inserted at the break point.
+    <dt><dfn>break-spaces</dfn></dt>
+    <dd>Line breaks are handled identically to ''overflow-wrap/normal'',
+      except that any sequence of <a>preserved</a> white space
+      that would otherwise overflow the line under these rules
+      must be broken after the last white space character that would fit the line.
```

Comment:
Sorry for the delayed answer.

If there is no spaces that would fit, you deal with it the way you would otherwise: break the line somewhere earlier if you can, or overflow if you cannot.

I don't actually have a strong opinion whether we should allow a break before the first space in a sequence of not, but I believe you were the one who expressed a preference for disallowing it.